jwql/website/manage.py

```diff
@@ -36,10 +36,25 @@
 import os
 import sys
 
+from jwql.utils.utils import get_config()
```

Comment:
I think this just needs to be `get_config` instead of a function call to `get_config()`

Reply:
100% correct. Change made.